### PR TITLE
Feature/minor improvements

### PR DESCRIPTION
**Pull request by davecarter**
Included in this PR:
- Updated to instantiable Rosetta.
- Added `react-addons-test-utils`.
- Updated test utilities.

### CODE REVIEW

**Comment by delacruz-dev** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -7,47 +7,47 @@ A generic links div container to fit as many content blocks with title and list
...
-    {
-      block: 'hamsters',
-      items: [
-        {literal: 'Syrian Hamsters', link: 'https://en.wikipedia.org/wiki/Golden_hamster'},
-        {literal: 'Dwarf Campbell', link: 'https://en.wikipedia.org/wiki/Campbell%27s_dwarf_hamster'},
-        {literal: 'Phodopus roborovskii', link: 'https://es.wikipedia.org/wiki/Phodopus_roborovskii'}
-      ]
-    }
-];
+This React component will iterate an array of objects to create different content block.
```
Comment (FranDelaCasa) [inline README.md]: ... to create different content blocks?

File: package.json [hunk 2]
```diff
@@ -33,27 +67,30 @@
...
     "react": "0.14.0"
   },
+"dependencies": {
+    "@schibstedspain/theme-basic": "latest"
+  },
   "devDependencies": {
     "@schibstedspain/frontend-pre-commit-rules": "latest",
-    "@schibstedspain/rosetta": "^2.1.0",
-    "@schibstedspain/theme-basic": "latest",
+    "@schibstedspain/rosetta": "latest",
```
Comment (FranDelaCasa) [inline package.json:75]: is Rosetta only for dev environment?

Comment (FranDelaCasa) [inline package.json:75]: :+1:

**Pull request merged**